apps/console/src/configs/routes.tsx

```diff
@@ -739,9 +742,11 @@ export const getAppViewRoutes = (): RouteInterface[] => {
             exact: true,
             icon: { icon: <EnvelopeGearIcon fill="black" className="icon" /> },
             id: "notificationChannels",
-            name: "Email & SMS",
+            name:  isPushProviderFeatureEnabled ? "Notification Channels" : "Email & SMS",
             order: 16,
-            path: `${ AppConstants.getDeveloperViewBasePath() }/email-and-sms`,
+            path: isPushProviderFeatureEnabled
+                ? AppConstants.getPaths().get("NOTIFICATION_CHANNELS")
+                : `${ AppConstants.getDeveloperViewBasePath() }/email-and-sms`,
```

Comment:
Shall we add this to the paths map here https://github.com/wso2/identity-apps/blob/58c52f689d687233389fcc0a8c979830339bf41c/features/admin.core.v1/constants/app-constants.ts#L265?